README.md

```diff
@@ -15,12 +15,16 @@ This project contains Terraform scripts to provision the Microsoft Azure Cloud i
 
 [<img src="./docs/images/viya4-iac-azure-diag.png" alt="Architecture Diagram" width="750"/>](./docs/images/viya4-iac-azure-diag.png?raw=true)
 
-This project helps you to automate the cluster-provisioning phase of SAS Viya deployment. To learn about all phases and options of the
-SAS Viya deployment process, see [Getting Started with SAS Viya and Azure Kubernetes Service](https://go.documentation.sas.com/doc/en/itopscdc/v_015/itopscon/n1d7qc4nfr3s5zn103a1qy0kj4l1.htm) in _SAS Viya Operations_.
+This project addresses the first of three steps in [Steps for Getting Started in SAS Viya Operations](https://go.documentation.sas.com/doc/en/itopscdc/v_031/itopscon/n12fgslcw9swbsn10rva4bp0mr2w.htm) in _SAS&reg; Viya&reg; Operations_:
+
```

Comment:
We intentionally removed these 2 links?   I thought they were nice to have. 
"To learn about all phases and options of the SAS Viya deployment process, see Getting Started with SAS Viya and Azure Kubernetes Service in SAS Viya Operations."
and "For more information about SAS Viya 4 requirements and documentation for the deployment process, refer to the SAS® Viya® 4 IT Operations Guide."? 

 https://go.documentation.sas.com/doc/en/itopscdc/v_031/itopscon/n12fgslcw9swbsn10rva4bp0mr2w.htm
shouldn't we replace v_031 with default or do we really want to pin to that version?

Reply:
Yes--it did need to be "default." My bad! 
We don't necessarily need that additional link to the documentation because the customer has most likely seen the doc--it's accessible from a link in the SOE.

Reply:
Thanks for updating that Getting Started link using "default". 

I agreed that we don't need the link to SAS® Viya® 4 IT Operations Guide because like you said there is a link in SOE directly point to that page.  But I think it would be nice if we keep the direct link to the SAS Viya and Microsoft Azure page, https://go.documentation.sas.com/doc/en/itopscdc/default/itopscon/n1d7qc4nfr3s5zn103a1qy0kj4l1.htm.  It complements this readme.

Reply:
Fine with me. Restored that reference.